modules/nw-kuryr-cleanup.adoc

```diff
@@ -46,13 +49,13 @@ $ source /tmp/venv/bin/activate
 +
 [source,terminal]
 ----
-(venv) $ pip install pip --upgrade
+(venv) $ pip install --upgrade pip
 ----
 .. Install the required Python packages by running the following command:
 +
 [source,terminal]
 ----
-(venv) $ pip install openstacksdk==0.54.0 python-openstackclient==5.5.0 python-octaviaclient==2.3.0
+(venv) $ pip install openstacksdk==0.54.0 python-openstackclient==5.5.0 python-octaviaclient==2.3.0 'python-neutronclient<9.0.0'
```

Comment:
BTW, should a note be added why we are preventing installing python-neutronclient in version 9 or higher?

Reply:
If it's helpful context, sure.

Reply:
Added comment on this in the note section.